requirements.txt

```diff
@@ -1,5 +1,6 @@
 deepspeech==0.7.1
 google-cloud-speech==1.3.2
+tensorflow
```

Comment:
This needed?

Reply:
Unfortunately yes...
```
$ python download.py -s 1 --use-cache

Traceback (most recent call last):
  File "download.py", line 15, in <module>
    from utils import convert_to_bool
  File "/mnt/c/Inn/Github/tarteel-ml/utils/__init__.py", line 3, in <module>
    from .audio import *
  File "/mnt/c/Inn/Github/tarteel-ml/utils/audio.py", line 14, in <module>
    import tensorflow as tf
ModuleNotFoundError: No module named 'tensorflow'
```

I think sooner or later they will install tensorflow

Do you prefer that I fix the code instead?

Reply:
ok, this is fine, I would prefer to pin this to a version

Reply:
looks like `2.5.0` is the latest

Reply:
added :)